### ISSUE TITLE
site footer links -- swamp left & order

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-website#107]
In thinking about people finding translations and about screen mag users... I'm thinking that in the footer it would better to put the links on the left (with probably a 4.5:1 vertical bar separating the sections).

Also re-thinking order:

* [icon] Translations
* Resources for Roles
* News
* Site Map
* Archive
* Contact WAI

**Comment by yatil** [comment]
Slightly concerned about having the translations icon on the same page for two different meanings: Open language selection and general information about translations.

I also think putting the link list to the left will unbalance the page. There will be a lot of space between the links and the rest of the footer, so unsure about the need for a separator line:

![screenshot 2018-10-19 at 09 37 50](https://user-images.githubusercontent.com/58862/47204108-c3dc3d00-d382-11e8-8b4c-84336d5ca97b.png)

@slhenry I can make the change once you holler, just let me know if you want a separator line with the mockup above. (I guess we want to add an icon anyway only after we have decided on one.)

**Comment by slhenry** [comment]
Thanks!

I agree that doesn't look good. Let's just leave this open for now and later we can re-think it.

**Comment by yatil** [comment]
I would like to close this issue for now. If we have further ideas for the footer, let‘s open a new issue. OK?
